packages/editor/src/components/editor-notices/style.scss

```diff
@@ -34,5 +34,20 @@
 
 .components-editor-notices__snackbar {
 	bottom: 20px;
-	left: 20px;
+	display: flex;
+	justify-content: center;
+	width: 100%;
+	// Width needs to accommodate sidebar for centred positioning to work across all screens.
+	@include break-medium() {
+		width: calc(100% - #{ $admin-sidebar-width-big });
+
+		body.folded &,
+		body.auto-fold & {
+			width: calc(100% - #{ $admin-sidebar-width-collapsed });
+		}
+	}
+
+	@include break-large() {
+		width: calc(100% - #{ $admin-sidebar-width });
+	}
```

Comment:
Ideall,y the styles that are specific to "WP Admin" should move to this stylesheet `packages/edit-post/src/components/layout/style.scss` since the styles in this file can be used in any context (think a page where there's not sidebar...)

Reply:
👍 That's really helpful to know—I was wondering why there were styles for this component outside of the component itself. It makes things a bit confusing to work with. I'll try to move anything that refers to the sidebar over to that file.